Classes/Trait/QueryBuilderTrait.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the package jweiland/service-bw2.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace JWeiland\ServiceBw2\Trait;
```

Comment:
Traits

Trait is a php reserved keyword

Reply:
Aww!! How I missed that ;)